docs/api.yaml

```diff
@@ -0,0 +1,844 @@
+openapi: "3.0.3"
+info:
+  #  description:
+
+  version: "1.0.0"
+  title: "HEI Admin API"
+  contact:
+    email: "contact@hei.school"
+
+  #servers:
+  # - description: To complete
+  #   url: example.com
+
+paths:
+  /grades:
+    post:
+      tags:
+        - "Grade"
+      summary: "Create new grades for a student's enrolled courses"
+      description: ""
+      operationId: "createGrade"
+      requestBody:
+        description: "The grade to attribute to the student and his enrolled course"
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateGrade'
+      responses:
+        200:
+          description: "The created grade"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Grade'
+        400:
+          $ref: "#/components/responses/400"
+        403:
+          $ref: "#/components/responses/403"
+        404:
+          $ref: "#/components/responses/404"
+        429:
+          $ref: "#/components/responses/429"
+        500:
+          $ref: "#/components/responses/500"
+
+  /students:
+    get:
+      tags:
+        - "Student"
+      summary: "Find students with optionnal search criteria"
+      parameters:
+        - name: "id"
+          in: "query"
+          description: "Identifier of the student to find"
+          required: false
+          schema:
+            type: string
+        - name: from
```

Comment:
These from/to parameters are not relevant in this API. What we might want is some parameter for filtering the timetable attribute, as it will grow through time and will eventually become huge. We can imagine some attribute like `futureTimetableOnly: boolean` or  `timetableForCourses: list_of_courses_uuid`. But let's wait for the need to arise. For the moment, just remove from/to.